src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -254,6 +256,25 @@ private Step CreateStep(Step jsonStepInfo, HostControlInfo hostControlInfo, int
         private void Popup_StepClosed(string name, Step.StepTypes stepType)
         {
             GuideFlowEvents.OnGuidedTourFinish(currentGuide.Name);
+
+            //The exit tour popup will be shown only when a popup (doesn't apply for survey) is closed or when the tour is closed. 
+            if(stepType != Step.StepTypes.SURVEY)
+                CreateRealTimeInfoWindow();
+        }
+
+        private void CreateRealTimeInfoWindow()
+        {
+            //Search a UIElement with the Name "statusBarPanel" inside the Dynamo VisualTree
+            UIElement hostUIElement = Guide.FindChild(mainRootElement, "statusBarPanel");
+
+            //Creates the RealTimeInfoWindow popup and set up all the needed values to show the popup over the Dynamo workspace
+            var exitTourPopup = new RealTimeInfoWindow();
+            exitTourPopup.VerticalOffset = 30;
+            if (hostUIElement != null)
+                exitTourPopup.PlacementTarget = hostUIElement;
+            exitTourPopup.Placement = PlacementMode.Center;
+            exitTourPopup.TextContent = Res.ExitTourWindowContent;
```

Comment:
I feel like we can use a constructor to pass in this? Or at least this can be set when constructing it above?
```
var exitTourPopup = new RealTimeInfoWindow()
{
  TextContent = Res.ExitTourWindowContent;
};
```

Reply:
@QilongTang the construction of the RealTimeInfoWindow object was updated using the syntax described by you, just some properties are assigned after construction.
see commit: https://github.com/DynamoDS/Dynamo/pull/11918/commits/cc26a3d36d1f186cbc0d598fc4e1c1d3f1d42b31